src/components/ui/ResultCount.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+import React from 'react'
+import isNumber from 'lodash.isnumber'
+
+const cmz = require('cmz')
+
+const cx = {
+  result: cmz(`
+    font-size: 1.0625rem;
+  `),
+
+  resultNumber: cmz(`
+    font-weight: 600;
+    margin-right: 5px;
+  `)
+}
+
+type Props = {
+  items?: number
+}
+
+const ResultCount = ({ items }: Props) => {
+  const countItems = isNumber(items) ? items : 0
+  const resultText = parseInt(countItems, 10) > 1 ? 'results' : 'result'
```

Comment:
I suggest to change the condition here because in case of `0` it should be plural as well, not singular as it is now:
```suggestion
  const resultText = parseInt(countItems, 10) !== 1 ? 'results' : 'result'
```

![image](https://user-images.githubusercontent.com/579331/54030258-0d0b1d00-41bc-11e9-9fd7-ac0eb92ccb4d.png)